### PR TITLE
Reworked the Whole Testing Chapter

### PR DESCRIPTION
**Pull request by jGleitz**
I added few new content but streamlined the wording and organised the points. 

Note that the chapter still needs **careful review**!

### CODE REVIEW

**Comment by jGleitz** [comment]
fixes #183

**Comment by acBerger** [comment]
[PDF] (http://jgleitz.github.io/Beagle/branches/revise-testing/Requirements%20Specification.pdf)

**Comment by jGleitz** [comment]
I did not spend as much time on this as I wish I could have. So please proof-read carefully!

**Comment by acBerger** [comment]
lgtm

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 1]
```diff
@@ -822,67 +919,93 @@ gls{Beagle}
...
 \end_inset
 
- does not change source code files and other input except from the 
-\begin_inset ERT
-status open
-
-\begin_layout Plain Layout
+ shuts down the computer if requested by a manual test.
+ First, run a measurement and activate shutting down and then check if it
+ worked and the results are saved.
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: were saved

File: doc/Requirements Specification/Parts/Test Cases.lyx [hunk 2]
```diff
@@ -899,18 +1022,22 @@ gls{Beagle}
...
-reference "/OF30/"
+reference "/Q10/"
 
 \end_inset
 
- and 
+.
+ A new system has to be set up with only these software applications and
+ a reasonable number of the functionality tests have to be run on it (may
+ for example be achieved by running all tests on a continuous integration
```
Comment (RomanLangrehr) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: I think there should be commas around `for example`

File: doc/Requirements Specification/Parts/Test Cases.lyx (hunk 1 above)
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: Fixed it, thank you!

File: doc/Requirements Specification/Parts/Test Cases.lyx (hunk 2 above)
Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: I don’t think so, but I’m not sure. @m1cm1c?

**Comment by RomanLangrehr** [comment]
:+1:

File: doc/Requirements Specification/Parts/Test Cases.lyx (hunk 2 above)
Comment (m1cm1c) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: I think they're optional.

Comment (jGleitz) [inline doc/Requirements Specification/Parts/Test Cases.lyx]: > I think they're optional.

That’s what I think. And they interrupt the flow of reading. So I vote for leaving them out.

**Comment by flyfl** [comment]
Lgtm

**Pull request merged**